index.html

```diff
@@ -77,14 +77,14 @@
           </div><!-- /.col -->
         </div><!-- /.row -->
         <div class="row justify-content-center ">
-          <div class="card shadow-lg rounded " style="width: 65rem;">
```

Comment:
rem is a flexible unit, these all px values at each div are just holding the card together which might break anytime something is worked upon the about card.
Try to look for the cause of the about card getting shrinked.